Sources/AIProxy/Fal/FalRunwalGen3AlphaInputSchema.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  FalRunwayGen3AlphaInputSchema.swift
+//
+//
+//  Created by Todd Hamilton on 10/4/24.
+//
+
+import Foundation
+
+/// Input schema for Runway Gen3 Alpha image-to-video model
+/// Docstrings from:
+/// https://fal.ai/models/fal-ai/runway-gen3/turbo/image-to-video/api
+public struct FalRunwayGen3AlphaInputSchema: Encodable {
+    // Required
+
+    /// The prompt to use for generating the video
+    public let prompt: String
```

Comment:
Swap with line 20 so that all required members are alphabetical. The pattern I've been using is to have both the 'required' section and 'optional' section sorted alphabetically. You could argue here that it's clearer to thematically order the properties. The reason I'm avoiding that is because as this lib grows different contributors will make different judgement calls about what is a thematic grouping or ordering. So, I've landed on the approach of keeping all properties alphabetical, even when it splits up things like `width` and `height`